pkg/kubelet/kubelet.go

```diff
@@ -249,12 +250,13 @@ type Dependencies struct {
 
 // makePodSourceConfig creates a config.PodConfig from the given
 // KubeletConfiguration or returns an error.
-func makePodSourceConfig(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Dependencies, nodeName types.NodeName) (*config.PodConfig, error) {
+func makePodSourceConfig(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Dependencies, nodeName types.NodeName) (*config.PodConfig, checkpoint.Manager, error) {
```

Comment:
this is pretty strange to return the checkpoint manager from the makePodSourceConfig func. Also the comment is not accurate anymore.